packages/components/collapsible-panel/src/header-icon.tsx

```diff
@@ -39,11 +55,12 @@ const HeaderIcon = (props: THeaderIcon) => {
           width: ${props.size === 'small'
             ? sizeIconContainerSmall
             : sizeIconContainer};
-          border-radius: 50%;
-          flex-shrink: 0;
-          box-shadow: ${designTokens.shadow7};
-          background-color: ${backgroundColor};
-          border: 1px solid ${backgroundColor};
+
+          border-radius: ${checkIfTestOrDefault(theme, '50%')};
+          flex-shrink: ${checkIfTestOrDefault(theme, '0')};
+          box-shadow: ${checkIfTestOrDefault(theme, designTokens.shadow7)};
+          background-color: ${checkIfTestOrDefault(theme, backgroundColor)};
+          border: ${checkIfTestOrDefault(theme, backgroundColor)};
```

Comment:
Perhaps if it's not possible to do with design tokes we could add those styles as a separate array item:
```js
[
...,
theme === 'default' &&
  css`
    border-radius: 50%;
    flex-shrink: 0;
    box-shadow: ${designTokens.shadow7};
    background-color: ${backgroundColor};
    border: 1px solid ${backgroundColor};
  `,
...
]
```

Reply:
Thank you very much @kark , I am going to update soon.

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2354/commits/a1d3831fa56c3c1c8ec1f3ba0f4a5e3cc666c4bf

Thank you.